services/QuillLMS/engines/evidence/spec/controllers/evidence/activities_controller_spec.rb

```diff
@@ -116,6 +116,44 @@ module Evidence
         expect(parent_activity.id).to(eq(new_activity.parent_activity_id))
         expect(new_activity.present?).to(eq(true))
       end
+
```

Comment:
The formatting of all three post requests can be cleaned up for readability.  Here's the first one:
```
post :create, 
  params: {
    activity:  {
      parent_activity_id: activity.parent_activity_id,
      scored_level:  activity.scored_level, 
      target_level: activity.target_level, 
      title: activity.title, 
      notes: activity.notes, 
      prompts_attributes: [{ text: "meat is bad for you.", conjunction: "because"}]
    }
  }
 ```